src/views/schedule/Schedule.tsx

```diff
@@ -0,0 +1,135 @@
+					<div className="schedule-countdown text-center">
+						<Countdown date={hackingBegins} />
+					</div>
+					<div className="schedule-countdown-text">
+						<span> Until Hacking Begins</span>
+					</div>
+				</section>
+			);
+		} else {
+			return (
+				<section id="schedule-block">
+					<div className="schedule-countdown text-center">
+						<Countdown date={devpostSubmission} />
+					</div>
+					<div className="schedule-countdown-text">
+						<span> Until Devpost Submission Closes</span>
+					</div>
+				</section>
+			);
+		}
+	}
+
+	return (
+		<Container>
+			<TitleBanner>
+				{generateCountdown()}
+			</TitleBanner>
+			{schedule.friday.map((event) =>
+				event.category === "Announcement" ? (
+					<Announcement
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						description={event.description}
+					/>
+				) : (
+					<Event
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						end={new Date(event.time.end).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						category={event.category}
+						host={event.host}
+						description={event.description}
+					/>
+				)
+			)}
+			{schedule.saturday.map((event) =>
+				event.category === "Announcement" ? (
+					<Announcement
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						description={event.description}
+					/>
+				) : (
+					<Event
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						end={new Date(event.time.end).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						category={event.category}
+						host={event.host}
+						description={event.description}
+					/>
+				)
+			)}
+			{schedule.sunday.map((event) =>
+				event.category === "Announcement" ? (
+					<Announcement
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						description={event.description}
+					/>
+				) : (
+					<Event
+						key={event.title}
+						title={event.title}
+						start={new Date(event.time.start).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						end={new Date(event.time.end).toLocaleTimeString([], {
+							hour: "2-digit",
+							minute: "2-digit",
+						})}
+						category={event.category}
+						host={event.host}
+						description={event.description}
+					/>
+				)
+			)}
```

Comment:
It seems we are repeating ourselves quite a lot here. If the schedule for each day would look the same, we could refactor this repeated logic.